src/node/constants.rs

```diff
@@ -43,3 +43,11 @@ pub const RENDER_NAME: &str = "renderD";
 /// Render DRM node prefix.
 #[cfg(target_os = "openbsd")]
 pub const RENDER_NAME: &str = "drmR";
+
+/// Directory to search DRM devices
+#[cfg(target_os = "openbsd")]
+pub const DRM_DIR_NAME: &str = "/dev";
+
+/// Directory to search DRM devices
+#[cfg(not(target_os = "openbsd"))]
+pub const DRM_DIR_NAME: &str = "/dev/dri";
```

Comment:
Should we use these constants in more places in `node/mod.rs`?

Reply:
The other place we use the path `/dev/dri` is for the function `dev_path` which is our equivalent to `drmGetMinorNameForFD`. Curiously the libdrm's function uses the constant everywhere. I will update the code to also make use of it.